include/pdal/util/Bounds.hpp

```diff
@@ -189,6 +202,10 @@ class PDAL_DLL BOX3D : private BOX2D
     BOX3D()
        { clear(); }
 
+    BOX3D(const BOX2D& box)
```

Comment:
I'd recommend making two constructors.  If you need to do some common stuff, just call a function.   You could also make the ctor explicit.  You just don't want that conversion to happen in most cases.